package.json

```diff
@@ -10,6 +10,12 @@
       "esm"
     ],
     "exports": {
+      ".": {
```

Comment:
This should just be 
```
".": "./src/session.ts",
```

(or `./src/index.ts` if you follow the other suggestion)

Reply:
done